sqswatcher/sqswatcher.py

```diff
@@ -206,6 +208,28 @@ def pollQueue(scheduler, q, t, proxy_config):
 
         time.sleep(30)
 
+
+def getInstanceMappingFile(region, proxy_config):
```

Comment:
We are doing it each time sqswatcher is starting up. Do you think we shouldn't be doing that? The race condition should not be a problem, because it the slot counts/ number of vcpus don't change that often and it is very rare that the file is being written by the jenkins task at the exact same time as it is being pulled from s3 bucket.